showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -98,7 +98,8 @@ class ShowkaseProcessor: AbstractProcessor() {
     private fun processComponentAnnotation(roundEnvironment: RoundEnvironment)
             : Set<ShowkaseMetadata> {
         val showkaseComposablesMetadata = processShowkaseAnnotation(roundEnvironment)
-        val previewComposablesMetadata = processPreviewAnnotation(roundEnvironment)
+//        val previewComposablesMetadata = processPreviewAnnotation(roundEnvironment)
```

Comment:
I did this just to be able to run the tests. The `processPreviewAnnotation` method crashes when running a test. This is because the `Class.forName(PREVIEW_CLASS_NAME)` throws a `ClassNotFoundException` :( I tried added the `RuntimeJar` to the classpath of androidTestImplementation(just like how I do it for testImplementation) but that doesn't solve the problem unfortunately!